Jenkinsfile

```diff
@@ -66,7 +67,7 @@ pipeline {
             }
         }
         stage('Linting & Doc checks') {
-            agent any
+            agent {label 'linux'}
```

Comment:
does this have to be linux now?

Reply:
I think this speeds up the build a bit since windows tend to be a bit slower and have a longer spawn time.